### PR TITLE
Replace Grunt with Webpack and add DEBUG builds

### PR DESCRIPTION
**Pull request by tri170391**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -94,6 +93,48 @@ deployment:
...
+            # Install debug streamroot-hlsjs-p2p-bundle
+            - rm -rf node_modules/streamroot-hlsjs-p2p-bundle
+            - npm install @streamroot/hlsjs-p2p-bundle-debug@beta
+            - mv node_modules/@streamroot/hlsjs-p2p-bundle-debug node_modules/streamroot-hlsjs-p2p-bundle
+            - mv node_modules/streamroot-hlsjs-p2p-bundle/hlsjs-p2p-bundle.js node_modules/streamroot-hlsjs-p2p-bundle/streamroot-hlsjs-p2p-bundle.debug.js
+
+            # Generate dist
+            - npm run build
+            - toolkit/add_banner.rb --file dist/$DIST_FILE
+                                    --deploy_env staging
```
Comment (maxbok) [inline circle.yml]: You should put `staging - debug` here.

File: circle.yml [hunk 2]
```diff
@@ -94,6 +93,48 @@ deployment:
...
+            #
+            ## End Debug builds
+            #
+
+            # Generate manifest
+            - toolkit/build_manifest.rb --name "Video.js 5 hls.js source handler"
+                                        --version $(toolkit/current_version.rb)
+                                        --platform "web"
+                                        --artifacts ./$DIST_FILE
+                                        --debug-artifacts   "https://circleci.com/api/v1/project/streamroot/$DIST_NAME/$CIRCLE_BUILD_NUM/artifacts/0/\$CIRCLE_ARTIFACTS/$DIST_NAME.debug.js"
```
Comment (maxbok) [inline circle.yml]: Too many spaces between `--debug-artifacts` and the url!!! 😛

File: circle.yml [hunk 3]
```diff
@@ -164,6 +205,50 @@ deployment:
...
+            # Install debug streamroot-hlsjs-p2p-bundle
+            - rm -rf node_modules/streamroot-hlsjs-p2p-bundle
+            - npm install @streamroot/hlsjs-p2p-bundle-debug@rc
+            - mv node_modules/@streamroot/hlsjs-p2p-bundle-debug node_modules/streamroot-hlsjs-p2p-bundle
+            - mv node_modules/streamroot-hlsjs-p2p-bundle/hlsjs-p2p-bundle.js node_modules/streamroot-hlsjs-p2p-bundle/streamroot-hlsjs-p2p-bundle.debug.js
+
+            # Generate dist
+            - npm run build
+            - toolkit/add_banner.rb --file dist/$DIST_FILE
+                                    --deploy_env preprod
```
Comment (maxbok) [inline circle.yml]: You should put `preprod - debug` here.

File: circle.yml [hunk 4]
```diff
@@ -164,6 +205,50 @@ deployment:
...
+            #
+            ## End Debug builds
+            #
+
+            # Generate manifest
+            - toolkit/build_manifest.rb --name "Video.js 5 hls.js source handler"
+                                        --version $(toolkit/current_version.rb)
+                                        --platform "web"
+                                        --artifacts ./$DIST_FILE
+                                        --debug-artifacts   "https://circleci.com/api/v1/project/streamroot/$DIST_NAME/$CIRCLE_BUILD_NUM/artifacts/0/\$CIRCLE_ARTIFACTS/$DIST_NAME.debug.js"
```
Comment (maxbok) [inline circle.yml]: Too many spaces between `--debug-artifacts` and the url!!! 😛

File: debug.html [hunk 5]
```diff
@@ -20,6 +20,8 @@
         <source src="http://sample.vodobox.net/skate_phantom_flex_4k/skate_phantom_flex_4k.m3u8" type="application/x-mpegURL"/>
         <!-- <source src="http://10.211.55.7/vod/hls_sample1_manifest.m3u8" type="application/x-mpegURL"/> -->
     </video>
+    <div id="streamroot-graphs"></div>
+    <script src="https://tools.streamroot.io/usage-graphs/latest/graphs.js"></script>
```
Comment (maxbok) [inline debug.html]: :+1:

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Dev]
+### Added
+- DEBUG builds.
```
Comment (jlouazel) [inline CHANGELOG.md]: This shouldn't be in the CHANGELOG. That doesn't change anything for the existing code and we shouldn't mention stuff we use on our side for debug purpose.

Comment (tri170391) [inline CHANGELOG.md]: :+1:

File: circle.yml (hunk 4 above)
Comment (tri170391) [inline circle.yml]: :+1:

File: circle.yml (hunk 3 above)
Comment (tri170391) [inline circle.yml]: :+1:

File: circle.yml (hunk 2 above)
Comment (tri170391) [inline circle.yml]: :+1:

File: circle.yml (hunk 1 above)
Comment (tri170391) [inline circle.yml]: :+1:

**Pull request merged**